docs/dashboard/catalog/products.md

```diff
@@ -208,14 +208,13 @@ Increase traffic to your store by adding SEO-friendly descriptions for search en
 
 Categorize your new product using the drop-down menus to assign a relevant Product Type and Category. This is mandatory before you can save the product. You can also optionally assign your product to a Collection.
 
-#### Step 5: Make your product visible
+#### Step 5: Make your product available
```

Comment:
Missing information. You need to set a category if you want to set the product as public.